src/components/Form/TimeSelector.js

```diff
@@ -20,9 +20,9 @@ import CardActions from "@material-ui/core/CardActions";
 import Select from "@material-ui/core/es/Select/Select";
 import InputLabel from "@material-ui/core/InputLabel";
 
-Date.daysBetween = function( date1, date2 ) {
+Date.daysBetween = function (date1, date2) {
```

Comment:
Try not to define functions on built-ins, instead make this a separate function.